use-cases/using-Intel-QAT-and-kata.md

```diff
@@ -0,0 +1,358 @@
+#Introduction
+
+Intel\* QuickAssist Technology (Intel\* QAT) provides hardware acceleration of security (cryptography) and compression. These instructions cover the steps for Clear Linux\* but can be adapted to any Linux\* distribution. Your distribution may already have the Intel QAT drivers, but it is likely they are out of date and do not contain the necessary user space components.  These instructions guide the user on how to download the kernel sources, compile kernel driver modules against those sources, and load them onto the host as well as preparing a specially built Kata Containers® kernel and custom Kata Containers rootfs.  
```

Comment:
- Please can you fold long lines like this as it makes diffs huge and difficult to read. We don't really have a document standard but something around 72-80 chars per line should work.
- s/acceleration of security/acceleration for security/ ?
- Please can you make "Clear Linux" a link to the website.

> but it is likely they are out of date and do not contain the necessary user space components.

Are we sure about this? I'd rather we state that CLR always has the latest so will provide the best experience and maybe point users off to launchpad/bugzilla/blah bugs for other distros if necessary.

Reply:
Clear probably has the latest driver but it is not that useful as it only includes kernel components. I'll remove the note about it being the latest.